documentation/modules/tracing/con-overview-opentracing-jaeger.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// assembly-distributed-tracing.adoc
+
+[id='con-overview-opentracing-jaeger-{context}']
+= Overview of OpenTracing and Jaeger
+
+Strimzi uses the OpenTracing and Jaeger projects.
+
+The OpenTracing specification defines APIs that developers can use to instrument applications for distributed tracing. It is independent from the tracing system.
```

Comment:
Just a thought. Would a couple of description lists work here?

Reply:
Nice idea. I'll have a go. We want to avoid overloading the user with info about OpenTracing and Jaeger.